ios/FluentUI.Demo/FluentUI.Demo/Demos/ColorDemoController.swift

```diff
@@ -124,13 +124,15 @@ class ColorDemoController: UIViewController {
         }
 
         NSLayoutConstraint.activate([
-            stackView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
-            stackView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
-            stackView.topAnchor.constraint(equalTo: view.topAnchor),
-            stackView.bottomAnchor.constraint(equalTo: view.bottomAnchor)
+            stackView.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor),
```

Comment:
do the cell look right on landscape? like selection of the cell background color and the separator goes edge to edge?

Reply:
Ah interesting, no. This constrains the scroll view to the safe areas in landscape, which means the separators don't go edge to edge. (No selection in this tableview, but if so it'd have the same problem.)

Do we have a pattern for this kind of situation? Without these changes the color picker is unusable:
![Simulator Screen Shot - iPhone 13 - 2021-10-01 at 15 35 59](https://user-images.githubusercontent.com/4934719/135693000-da161705-a4a7-4525-a9b0-aad79dcb60ea.png)

Reply:
we shouldn't be using stackview? but ultimately, can't we just fix our top constraint (maybe bottom constraint) to be within safearea?

Reply:
Somehow TableHeaderFooterView doesn't have this problem, but it's also a complicated mess that I don't necessarily want to spread. I'll just do your idea for now :(